app/assets/javascripts/application/evaluationtable.js

```diff
@@ -86,6 +86,25 @@ var evaluationTable = (function($) {
     localStorage.setItem('hiddenUsers', JSON.stringify(list));
   }
 
+  module.refreshSumOfPrinciple = function(input) {
+    var principle_id = input.getAttribute('data-principle');
+
+    total = calcSum(principle_id);
+
+    $('#sum-of-principle-' + principle_id).html(total);
```

Comment:
You should always avoid using `.html()`, because it is vulnerable to javascript injection. Use `.text()` instead.